src/components/FiltersDrawer/FiltersDrawer.tsx

```diff
@@ -166,22 +100,35 @@ export const FiltersDrawer = ({
         </Heading>
       </Drawer.Header>
       <Drawer.Body className={styles['filters-drawer__body']} ref={filtersBody}>
-        {filtersCheckboxFieldComponents}
+        {children}
       </Drawer.Body>
-      <Drawer.Footer className={footerClassName}>
-        <ButtonGroup className={styles['footer__button-group']}>
-          <Button className={styles['footer__button']} onClick={clearFilters}>
-            Clear All
-          </Button>
-          <Button
-            className={styles['footer__button']}
-            onClick={applyFilters}
-            variant="primary"
-          >
-            Apply
-          </Button>
-        </ButtonGroup>
-      </Drawer.Footer>
+      {(onClear || onApply) && (
```

Comment:
What if `onClear` or `onApply` is passed in but not both? What does the button without a method do?

Reply:
Dang, nice catch, was trying to make the individual buttons conditional as well, will fix